CHANGELOG.rst

```diff
@@ -2,6 +2,11 @@
 CHANGELOG
 =========
 
+1.2.dev1
+========
+
+* bug-fix: AmazonEstimators: fix valid range of hyper-parameter 'loss' in linear learner
+
 1.2.0
 ========
```

Comment:
This isn't your fault, but can you make sure the equal signs line up with the version.

1.2.0
====

Thanks!

Reply:
Thanks for pointing out.

Formats of change log fixed.